packages/node_modules/@webex/plugin-meetings/src/common/collection.js

```diff
@@ -0,0 +1,90 @@
+import {MEETINGS} from '../constants';
+
+/**
+ * @class Collection
+ */
+export default class Collection {
+  namespace = MEETINGS;
+
+  /**
+   * @param {String} property
+   * @memberof Collection
+   */
+  constructor(property) {
+    /**
+     * The property name for what collection will be stored, i.e., this.meetingInfos, this.meetings, etc.
+     * @instance
+     * @type {String}
+     * @public
+     * @memberof Meetings
+     */
+    this.propertyName = property;
```

Comment:
Is it worth throwing an error if no property name is passed?

Reply:
good point. added.